tests/models/test_cli.py

```diff
@@ -413,6 +415,30 @@ def test_prepare_env_fails(sk_model):
         assert p.wait() != 0
 
 
+@pytest.mark.parametrize("enable_mlserver", [True, False])
+def test_generate_dockerfile(sk_model, enable_mlserver):
+    with mlflow.start_run() as active_run:
+        if enable_mlserver:
+            mlflow.sklearn.log_model(
+                sk_model, "model", extra_pip_requirements=[PROTOBUF_REQUIREMENT]
+            )
+        else:
+            mlflow.sklearn.log_model(sk_model, "model")
+        model_uri = "runs:/{run_id}/model".format(run_id=active_run.info.run_id)
+    extra_args = ["--install-mlflow"]
+    if enable_mlserver:
+        extra_args.append("--enable_mlserver")
+
+    with tempfile.TemporaryDirectory() as temp_dir:
+        pyfunc_generate_dockerfile(temp_dir, model_uri, extra_args=extra_args)
+        output_directory = Path(temp_dir)
+        assert output_directory.is_dir()
+        assert output_directory.joinpath("Dockerfile").exists()
+        assert output_directory.joinpath("model_dir").is_dir()
+        # Assert file is not empty
+        assert output_directory.joinpath("Dockerfile").stat().st_size != 0
```

Comment:
```suggestion
    pyfunc_generate_dockerfile(tmp_path, model_uri, extra_args=extra_args)
    output_directory = Path(temp_dir)
    assert output_directory.is_dir()
    assert output_directory.joinpath("Dockerfile").exists()
    assert output_directory.joinpath("model_dir").is_dir()
    # Assert file is not empty
    assert output_directory.joinpath("Dockerfile").stat().st_size != 0
```

Reply:
@harupy 
I haven't found any fixtures for `tmp_path`
is it some default fixture in pyfunc?
I'm sorry if it is a stupid question -- I'm not experienced in Python at all

Reply:
> is it some default fixture in pyfunc?

Yes, it's provided by pytest.